backend/api/Startup.cs

```diff
@@ -166,10 +183,20 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseAuthorization();
 
             app.UseMiddleware(typeof(LogRequestMiddleware));
-
+            app.UseHealthChecks("/live", new HealthCheckOptions
```

Comment:
Is this now the actual route `/live` rather than `/liveliness`?  I don't understand the differences.  Also routes for the api need to be consistent `/api/...` or we'll have two different paths depending on the caller path.

Reply:
liveliness is not the route, it is just the name of the health check.
Step 1. Create health checks and add names/tags to them as necesssary.
Step 2. Map all health checks to endpoints based on the identifiers created in Step 1.